### PR TITLE
Version 0.19.0

### PR DESCRIPTION
**Pull request by yguclu**
* Improve developers' section of `README.rst`
* Remove broken `build status` badge
* Add new `CI status` badge which points to the workflows on the `master` branch
* Remove obsolete `.travis.yml` configuration file
* Increase library version to 0.19.0 in `pyproject.toml`

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -42,33 +43,37 @@ From sources
...
-must also be tested against the test suite of psydac. This can be done by opening
-a PR in psydac where the only change consists of installing the corresponding 
-branch of sympde.
+Because many important features of SymPDE are only tested in Psydac, new PRs should also be tested against the test suite of Psydac.
+This can be done by opening a PR in Psydac, where the only change consists of installing the corresponding branch of SymPDE.
+To achieve this, one just needs to modify the line corresponding to ``sympde`` in the ``pyproject.yaml`` file.
 
-For instance to test a new sympde branch called ``my_feature``, the following
-lines can be used in the file ``.github/workflows/continuous-integration.yml`` of psydac:
+For instance, to test a new SymPDE branch called ``my_feature``, one should write
```
[the remark this answers is not in the document]
Comment (yguclu) [reply README.rst:50]: Good point, let me update this.

[the remark this answers is not in the document]
Comment (yguclu) [reply README.rst:50]: Ah no sorry, this section is about properly testing new PRs in SymPDE. So writing ``master`` makes no sense here.

[the remark this answers is not in the document]
Comment (yguclu) [reply README.rst:50]: In other words, a new PR wants to merge a new branch called ``my_feature`` into ``master``, so ``my_feature`` is the branch that should be tested in the Psydac CI workflow.

File: README.rst [hunk 2]
```diff
@@ -42,33 +43,37 @@ From sources
...
+Also, pay attention to the words ``head`` and ``tags`` in the path: the former is used for Git branches, the latter is used for Git tags (which may or may not correspond to GitHub releases).
+
+
+.. |CI status| image:: https://github.com/pyccel/sympde/actions/workflows/continuous-integration.yml/badge.svg?branch=master&event=push
+    :alt: CI status
     :scale: 100%
-    :target:  https://travis-ci.com/pyccel/sympde
+    :target:  https://github.com/pyccel/sympde/actions/workflows/continuous-integration.yml
 
 .. |docs| image:: https://readthedocs.org/projects/sympde/badge/?version=latest
```
Comment (kvrigor) [inline README.rst:73]: Must be https://readthedocs.org/projects/docs/badge/?version=latest so that the docs badge shown is green.

Comment (yguclu) [reply README.rst:73]: I am not sure what the meaning of this badge is, as we do not have a documentation workflow AFAIK 😅 

These are the most recent builds: https://readthedocs.org/projects/sympde/builds/

Comment (yguclu) [reply README.rst:73]: And according to the documentation itself, the build is failing: https://readthedocs.org/projects/sympde/

Comment (kvrigor) [reply README.rst:73]: Right, existing link is already ok, we only just need to fix things at the backend (i.e. readthedocs webhook setting in GitHub). We can address this in the next release.

**Comment by kvrigor** [comment]
Everything LGTM 👍

**Review by kvrigor — APPROVED** [review]

**Pull request merged**